pulseapi/profiles/admin.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib import admin
+from .models import UserProfile, UserBookmarks
+
+class UserProfileAdmin(admin.ModelAdmin):
+    """
+    Show a list of entries a user has submitted in the EmailUser Admin app
```

Comment:
We'll change the comments in a future pr when we add the actual fields to a user profile?